app/main.ipynb

```diff
@@ -126,19 +155,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "subsequence = array[[1, 2, 5, 6]]\n",
+    "result = subsequence.sum()"
```

Comment:
The subsequence selected from the array does not sum up to 27. The correct indices should be chosen to achieve the desired sum. Please verify the indices used for multi-indexing to ensure the subsequence sums to 27.